lib/AppPluginZwave/index.js

```diff
@@ -1,251 +0,0 @@
-'use strict';
-
-/*
-	Plugin ID: zwave
-
-	This plugin installs homey-meshdriver.
-
-	Enable the plugin by adding `{ "id": "zwave" }` to your /.homeyplugins.json array
-
-	Plugin options:
-	{
-		"version": "latest"
-	}
-*/
-
-const fs = require('fs');
-const path = require('path');
-
-const fse = require('fs-extra');
-const fetch = require('node-fetch');
-const colors = require('colors');
-
-const AppPlugin = require('../AppPlugin');
-
-const NpmCommands = require('../Modules/NpmCommands');
-
-class AppPluginZwave extends AppPlugin {
-
-  async run() {
-    const appJson = await fse.readJSON(this._app._appJsonPath);
-    if (appJson.sdk === 3) {
-      return NpmCommands.install({ save: true }, {
-        id: 'homey-zwavedriver',
-        version: this._options.version,
-      });
-    }
-    return NpmCommands.install({ save: true }, {
-      id: 'homey-meshdriver',
-      version: this._options.version,
-    });
-  }
-
-	static createDriverQuestions() {
-		return [
-			{
-				type: 'confirm',
-				name: 'isZwave',
-				default: false,
-				message: 'Is this a Z-Wave device?'
-			},
-			{
-				type: 'input',
-				name: 'zwaveSigmaDetails',
-				message: 'Enter the Sigma ID (four digits, found in the URL) at ' + colors.underline('https://products.z-wavealliance.org/') + ' or press [return] to skip',
-				default: '',
-				when: answers => answers.isZwave,
-				filter: async input => {
-					if( input === '' ) return '';
-					let details = await AppPluginZwave.getSigmaDetails( input );
-						details.toString = () => details.Name
-					return details;
-				}
-			}
-		]
-	}
-
-	static async createDriver({ app, driverPath, answers, driverJson }) {
-
-		await app.addPlugin('zwave');
-    await NpmCommands.install({ save: true }, {
-      id: 'homey-meshdriver',
-    });
-
-		await fse.copy(
-			path.join(app.path, 'node_modules', 'homey-meshdriver', 'assets', 'driver', 'zwave', 'device.js'),
-			path.join(driverPath, 'device.js')
-		);
-		await fse.remove(
-			path.join(driverPath, 'driver.js')
-		);
```

Comment:
`AppPluginZwave#createDriver` installs homey-meshdriver and copies meshdriver templates